src/main/java/com/thevoxelbox/voxelsniper/command/executor/PaintExecutor.java

```diff
@@ -43,10 +49,15 @@ public void onPaintBack(
     @CommandMethod("<art>")
     public void onPaintArt(
             final @NotNull Sniper sniper,
-            final @Nullable @Argument("art") Art art
+            final @Nullable @Argument("art") String art
```

Comment:
That should have it's own argument / `VoxelCommandElement` ig - that was previously handled by the standardized Piston / WE EnumArgument